### PR TITLE
Added a new ebook

### PR DESCRIPTION
**Pull request by artorias111**
Hi! I added a new book to help in learning machine learning.

### CODE REVIEW

**Comment by Rishit-dagli** [comment]
@artorias111 Could you verify that this book is available to distribute freely?

Also, to move forward with this change could you sign the CLA mentioned in [this comment](https://github.com/Rishit-dagli/Python3-by-practice/pull/37#issuecomment-702675984)

**Comment by artorias111** [comment]
Hey, I'm sorry it appears this ebook is free only when the ebook is purchased, I'm afraid it's not allowed to be freely distributed. If it can't be added then go ahead and close the pr. Thanks!

**Comment by artorias111** [comment]
there is however, an IBM edition of the same book that's free to view and distribute online, shall I go ahead and make a new PR adding that?

**Comment by Rishit-dagli** [comment]
> there is however, an IBM edition of the same book that's free to view and distribute online, shall I go ahead and make a new PR adding that?

You can make that addition in this PR itself :)

**Comment by artorias111** [comment]
Oh alright sorry, Git newbie I wasn't aware of that. I made the changes thanks a lot!

File: Machine Learning/ibm-machine-learning-for-dummies-ibm-limited-edition_IMM14209USEN.pdf:Zone.Identifier [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[ZoneTransfer]
```
Comment (Rishit-dagli) [inline Machine Learning/ibm-machine-learning-for-dummies-ibm-limited-edition_IMM14209USEN.pdf:Zone.Identifier:1]: Could you remove the Identifier file?

**Review by Rishit-dagli — CHANGES_REQUESTED** [review]
Could you rename the PDF to `Machine-Learning-For-Dummies_IBM.pdf` to maintain consistency?

**Comment by artorias111** [comment]
Done

**Review by Rishit-dagli — APPROVED** [review]
This looks good to me, I will go ahead and merge this. Thank you for your contributions to this repo 💯 . Feel free to support this repo by leaving a 🌟

**Comment by Rishit-dagli** [comment]
@all-contributors please add @artorias111 for content

**Pull request merged**